cmd/kube-controller-manager/app/plugins.go

```diff
@@ -62,8 +62,9 @@ func ProbeAttachableVolumePlugins(config componentconfig.VolumeConfiguration) []
 	return allPlugins
 }
 
-// ProbeRecyclableVolumePlugins collects all persistent volume plugins into an easy to use list.
-func ProbeRecyclableVolumePlugins(config componentconfig.VolumeConfiguration) []volume.VolumePlugin {
+// ProbeControllerVolumePlugins collects all persistent volume plugins into an easy to use list.
+// Only provisioner/recycler/deleter volume plugins should be returned.
```

Comment:
unclear: does this filter for volumes that satisfy all three interfaces or any one of them?